wca-guidelines.md

```diff
@@ -201,12 +205,16 @@ To be more informative, each Guideline is classified using one of the following
 ## <article-E><fewest-moves><fewestmovessolving> Article E: Fewest Moves Solving
 
 - E2b+) [CLARIFICATION] A competitor may choose to stop an attempt early by handing in a solution before the time limit.
-- E2c+) [CLARIFICATION] Although the judge may provide a standard sheet for competitors to submit solutions, a competitor may submit a solution on a different piece of paper, as long as a single move sequence is unambiguously indicated as the submitted solution. (Note that the piece of paper must come from the judge, according to [Regulation E3a](regulations:regulation:E3a).)
-- E2c++) [CLARIFICATION] The competitor's solution must only use moves that are exactly defined in [Regulation 12a](regulations:regulation:12a). Examples of notation and moves that are not permitted in competition: [F], [R, U], [R: U], R'2, L'w, f, M, U2'.
+- E2c+) [CLARIFICATION] Although the judge may provide a standard sheet for competitors to submit solutions, a competitor may submit a solution on a different piece of paper. (Note that the piece of paper must come from the judge, according to [Regulation E3a](regulations:regulation:E3a).)
+- E2c1+) [ADDITION] If the competitor forgot to write their name on their intended solution paper before the end of 60 minutes, they may be allowed to add their name (without modifying their solution or making any other changes) under direct supervision of the judge, at the discretion of the Delegate. The competitor must ask and write their name before they finish submitting their solution (e.g. placing it in the judge's hands and allowing the judge to move on to the next competitor, placing it on the judge's table and walking away), and are not allowed to add their name later.
+- E2c2+) [EXAMPLE] Examples that are not considered a single unambiguous move sequence written out as one or multiple horizontal lines of text: a solution with extra moves, multiple solution sequences without exactly one boxed/circled solution, a solution with ambiguous symbols (e.g. a letter that looks similar to "B" and "R" but is not clearly one of them), a solution written in a spiral, a solution with stars or arrows indicating moves to be performed out of written order (e.g. "insertions" that are not written in-line with the rest of the move sequence, "pre-moves" that are not incorporated into the move sequence using valid notation), any arrangement of symbols that cannot be interpreted as an unambiguous sequence of "top-to-bottom" text lines with each line in "left-to-right" reading order.
+- E2c2++) [CLARIFICATION] The submitted page may contain written moves on both sides, as long only one side has a boxed/circled solution.
```

Comment:
Should this read "as long AS only one side..."?

Reply:
Yep, thanks!